docs/book/src/forc/workspaces.md

```diff
@@ -25,6 +26,27 @@ members = ["member1", "path/to/member2"]
 The `members` field accepts entries to be given in relative path with respect to the workspace root.
 Packages that are located within a workspace directory but are *not* contained within the `members` set are ignored.
 
+## The `[patch]` section
+
+The `[patch]` section can be used to override any dependency in the workspace dependency graph. The usage is the same with package level `[patch]` section and details can be seen [here](./manifest_reference.md#the-patch-section).
+
+If a member of a workspace has a patch table in its package manifest, the workspace level patch declarations are merged with package level declarations. If there are conflicting declarations (patching the same source with different target) between workspace level patch table and a member's patch table, workspace level patch declarations.
+
+> **NOTE:** Workspace level patch declarations only apply to members declared in `members` field.
```

Comment:
Just to clarify, does this note refer to the current `[patch]`-table-merging behaviour, or the application of patches?

Patches from a `[patch]` table should be applied to *all* occurrences of the specified package within the package graph (not only to the top-level members). I *think* we already do this in our package `[patch]` table support, but just want to clarify.

Reply:
Yeah exactly, we already do this for workspace patch table as well but the wording is pretty confusing. Removed it